styles/styles.css

```diff
@@ -156,48 +155,153 @@ a:hover {
-  padding: 0.5em 1.2em;
+  min-height: 3rem;
+  padding: 0.5rem 1.25rem;
+  border: none;
+  border-radius: 1.5rem;
+  background-color: transparent;
+  color: inherit;
   font-family: var(--body-font-family);
+  font-size: var(--spectrum-font-size-300);
   font-style: normal;
-  font-weight: var(--spectrum-medium-font-weight);
-  line-height: 1.25;
+  font-weight: var(--spectrum-bold-font-weight);
+  line-height: 1.3;
   text-align: center;
   text-decoration: none;
-  background-color: var(--color-background-button);
-  color: var(--color-text-button);
-  cursor: pointer;
   overflow: hidden;
-  text-overflow: ellipsis;
   white-space: nowrap;
+  text-overflow: ellipsis;
+  cursor: pointer;
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:disabled,
+  &:disabled:hover {
+    color: var(--color-text-button-disabled);
+    cursor: unset;
+  }
+
+  &:focus,
+  &:focus-visible {
+    outline: 2px solid var(--spectrum-focus-indicator-color);
+    outline-offset: 2px;
+  }
 }
 
-a.button:hover,
-a.button:focus,
-button:hover,
-button:focus {
-  background-color: var(--color-background-button-hover);
-  cursor: pointer;
+.button--primary {
+  --color-button-primary-background: rgb(from var(--spectrum-black) r g b / 84%);
+  --color-button-primary-background-hover: rgb(from var(--spectrum-black) r g b / 93%);
+  --color-button-primary-background-disabled: rgb(from var(--spectrum-black) r g b / 9%);
+  --color-text-button-primary: var(--spectrum-white);
+  
+  background-color: var(--color-button-primary-background);
+  color: var(--color-text-button-primary);
+
+  &:hover {
+    background-color: var(--color-button-primary-background-hover);
+    color: var(--color-text-button-primary);
+  }
+
+  &:disabled,
+  &:disabled:hover {
+    background-color: var(--color-button-primary-background-disabled);
+  }
 }
 
-button:disabled,
-button:disabled:hover {
-  opacity: var(--spectrum-opacity-disabled);
-  cursor: unset;
+:root:has(#color-scheme option[value="dark"]:checked) .button--primary {
+  --color-button-primary-background: rgb(from var(--spectrum-white) r g b / 85%);
+  --color-button-primary-background-hover: rgb(from var(--spectrum-white) r g b / 94%);
+  --color-button-primary-background-disabled: rgb(from var(--spectrum-white) r g b / 11%);
+  --color-text-button-primary: var(--spectrum-gray-25);
 }
 
-a.button.secondary,
-button.secondary {
-  background-color: unset;
-  border: 2px solid currentcolor;
-  color: var(--color-text);
+@media (prefers-color-scheme: dark) {
+  :root:has(#color-scheme option[value="system"]:checked) .button--primary {
+    --color-button-primary-background: rgb(from var(--spectrum-white) r g b / 85%);
+    --color-button-primary-background-hover: rgb(from var(--spectrum-white) r g b / 94%);
+    --color-button-primary-background-disabled: rgb(from var(--spectrum-white) r g b / 11%);
+    --color-text-button-primary: var(--spectrum-gray-25);
+  }
+}
+
+.button--outline {
```

Comment:
Can we rename the button themes to match Spectrum? 
https://opensource.adobe.com/spectrum-css/?path=/docs/components-button--docs

Accent, Primary, Static Black, Static White

Reply:
With the Primary having Filled and Outline variants, they should now be as follows:
- primary
- primary-outline
- accent
- static-black (added)
- static-white (added)
- ghost